doc/Programming-guide/Key-platform-features/Data-collection/index.md

```diff
@@ -5,4 +5,1175 @@ permalink: /:path/
+
+* ### [Flume log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Flume-log-appender)
+
+The Flume log appender encapsulates received logs into Flume events and sends these events to external Flume sources via Avro RPC. You can configure
+flume event format, hosts balancing type, include client-side and/or server-side profile data and
+[other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/#configuration) parameters for this log appender.
+
+* ### [Kafka log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Kafka-log-appender)
+
+The Kafka log appender is responsible for transferring logs from the Operations service to the [Apache Kafka](http://kafka.apache.org/) service. The logs are
+stored in the specified topic. You can configure Kafka bootstrap servers list, topic name, compression type and
+[other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Kafka-log-appender/#configuration) parameters for this log appender.
+
+* ### [MongoDB log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/MongoDB-log-appender)
+
+The MongoDB log appender is responsible for transferring logs from the Operations service to the [MongoDB](https://www.mongodb.com/) database. You can
+configure MongoDB nodes list, authentication credentials, MongoDB database name and
+[other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/MongoDB-log-appender/#configuration) parameters for this log appender.
+The logs are stored in the table named __logs_$[applicationToken]({{root_url}}Glossary)_, where _$[applicationToken]({{root_url}}Glossary)_ matches the token
+of the current application.
+
+* ### [Oracle NoSQL log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Oracle-NoSQL-log-appender)
+
+The Oracle NoSQL log appender is responsible for transferring logs from the Operations service to the
+[Oracle NoSQL](http://www.oracle.com/technetwork/database/database-technologies/nosqldb/overview/index.html) key/value storage. You can configure KVStore name,
+KVStore nodes and [other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Oracle-NoSQL-log-appender/#configuration) parameters for this
+log appender.
+
+* ### [Rest log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Rest-log-appender)
+
+The REST log appender is responsible for transferring logs from Operation service to your custom service. You can configure host, port, relative URI path,
+method type and [other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Rest-log-appender/#configuration) parameters for this log appender.
+
+## Custom appender implementations
+
+Refer to the [Custom log appender]({{root_url}}Customization-guide/Customizable-system-components/Log-appenders) page to learn how to implement custom log
+appenders.
+
+## Managing log appenders
```

Comment:
This section is a part of 'Custom appender implementations' and of introduction to the log appenders as well. It's text does not relate to the heading. Could this be removed (probably with updates to the respective parts)?